modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -1,38 +1,127 @@
-= Controlling File Versions and Aging
+= Controlling File Versions
 
-The Versions app (files_versions) expires old file versions
-automatically to ensure that users don’t exceed their storage quotas.
-This is the default pattern used to delete old versions:
+== How Versions are created
 
-* For the first second we keep one version
-* For the first 10 seconds ownCloud keeps one version every 2 seconds
-* For the first minute ownCloud keeps one version every 10 seconds
-* For the first hour ownCloud keeps one version every minute
-* For the first 24 hours ownCloud keeps one version every hour
-* For the first 30 days ownCloud keeps one version every day
-* After the first 30 days ownCloud keeps one version every week
+Every time when a file is written to the storage the versions app
+(files_versions) creates a new backup copy of the file. These backups are
+stored inside a folder `files_versions` inside the users root folder.
+The app will add the suffix `.v` followed by the unix timestamp of the creation
+date of the backup copy.
```

Comment:
pls clarify if this is also true on external storage mounts

Reply:
@cdamken @pako81 @dercorn any ideas?

Reply:
I am pretty sure that external wnd shares will not create new files versions when the file is changed on the Windows Server

Reply:
OK, tested it.
If you modify a file lying on an external storage from owncloud, it will generate a file version.
BUT if the file is changed on the Windows side, it will not generate a new version.

That is what I expected and how it its designed.
External Storage has many limitations.

Reply:
Should we document it? This belongs to WND

Reply:
I would drop a note about like:
NOTE: File versioning only gets triggered if the change is made via the ownCloud ecosystem. It does not get triggered if the change is made at a mounted filesystem directly.

Reply:
ok, done

Reply:
done

Reply:
done